### PR TITLE
Add RaspberryPi GPIO plugin

### PR DESCRIPTION
**Pull request by mhall119**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -191,6 +191,7 @@ github.com/google/go-querystring v1.0.0/go.mod h1:odCYkC5MyYFN7vkCjXpyrEuKhc/BUO
 github.com/google/gofuzz v0.0.0-20161122191042-44d81051d367/go.mod h1:HP5RmnzzSNb993RKQDq4+1A4ia9nllfqcQFTQJedwGI=
 github.com/google/gofuzz v1.0.0 h1:A8PeW59pxE9IoFRqBp37U+mSNaQoZ46F1f0f863XSXw=
 github.com/google/gofuzz v1.0.0/go.mod h1:dBl0BpW6vV/+mYPU4Po3pmUjxk6FQPldtuIdl/M65Eg=
+github.com/google/martian v2.1.0+incompatible h1:/CP5g8u/VJHijgedC/Legn3BAbAaWPgecwXBIDzw5no=
```
Comment (ssoroka) [inline go.sum]: please run `go mod tidy`

Comment (mhall119) [reply go.sum]: Rebasing off the latest master branch removes this from my PR

**Comment by danielnelson** [comment]
I haven't looked in depth, but these bits in the dependency are an issue since it is called during package initialization:

https://github.com/stianeikeland/go-rpio/blob/a36b96d0b1a40b37017d64875275e7e6312408c7/rpio.go#L109
https://github.com/stianeikeland/go-rpio/blob/a36b96d0b1a40b37017d64875275e7e6312408c7/rpio.go#L719

Overall, I think this would be a good candidate to begin life as an execd script, and we can consider building it in later.

**Comment by sjwang90** [comment]
@mhall119 Are you still planning on submitting this as an external plugin?

**Comment by srebhan** [comment]
Hey @mhall119 what is the state of this PR?

**Comment by srebhan** [comment]
@mhall119 do you have any plans to work on this PR?

**Comment by sjwang90** [comment]
Going to close this - @mhall119 Feel free to re-open if you'd like to continue working on this.

**Pull request closed without merging**